boards/st/stm32mp135f_dk/stm32mp135f_dk.dts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/mp13/stm32mp135.dtsi>
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <st/mp13/stm32mp135faex-pinctrl.dtsi>
```

Comment:
for clarity (and consistency with other STM boards), include the dt-bindings header file related to this very DTS file below the DTSI's inclusions:
```c
#include <st/mp13/stm32mp135.dtsi>
#include <st/mp13/stm32mp135faex-pinctrl.dtsi>
#include <zephyr/dt-bindings/input/input-event-codes.h>
```